Sources/SwiftFrameCore/Workers/ConfigProcessor.swift

```diff
@@ -110,14 +99,17 @@ public class ConfigProcessor: VerbosePrintable {
 
         try deviceData.screenshotsGroupedByLocale.forEach { locale, imageDict in
             group.enter()
+            defer { group.leave() }
```

Comment:
Wait what this works 🤯  I always thought one can only do this for a method, not a loop

Reply:
> A `defer` statement is used for executing code just before transferring program control outside of the scope that the statement appears in.

Honestly I didn't think about it when I wrote it, the compiler didn't complain 😄 but I verified that it's not accidentally using method-scope but the for-loop-scope with this code:

```swift
func deferTest() {
    let values = [1, 2, 3, 4]
    for value in values {
        defer { print(value) }
        print("Loop")
    }
}
```

prints:

```
Loop
1
Loop
2
Loop
3
Loop
4
```

Reply:
🤯  the more you know